### DISCUSSION TITLE
Documentation that lists needed Internet access

### DISCUSSION THREAD

**Discussion by lonelyadmin** [Security-Onion-Solutions/securityonion#2203] [Unsupported Versions] [answered]
Is there any documentation on what external internet access is needed for each role or service?  

I ran across some firewall logs showing the (so-)launcher connecting to some random googleusercontent site and wasn't sure if it was legit.

Running latest 2.3 from iso.

**Comment by weslambert** [comment, marked as answer]
Hi @lonelyadmin,  we have an issue in to track this here: https://github.com/Security-Onion-Solutions/securityonion/issues/2046
